HLTrigger/JetMET/plugins/HLTJetTimingFilter.h

```diff
@@ -54,6 +55,7 @@ HLTJetTimingFilter<T>::HLTJetTimingFilter(const edm::ParameterSet& iConfig)
           consumes<edm::ValueMap<float>>(iConfig.getParameter<edm::InputTag>("jetEcalEtForTiming"))},
       minJets_{iConfig.getParameter<unsigned int>("minJets")},
       jetTimeThresh_{iConfig.getParameter<double>("jetTimeThresh")},
```

Comment:
A clearer name for this parameter would have been `jetMinTimeThresh` (was missed in #35724), especially after this PR.

On the other hand, backporting this renaming to `13_0_X` would introduce a non-backward compatible change (to the module's PSet) in a closed release, which I would avoid. (and doing the renaming only in 13_1_X, but not in 13_0_X, makes it harder for TSG to migrate menus from `13_0_X` to `13_1_X`)

Unless there are suggestions, we'll open a GitHub issue to improve the parameter names in the future (without renaming in this PR).